fluss-connectors/fluss-connector-flink/src/main/java/com/alibaba/fluss/connector/flink/sink/FlinkSink.java

```diff
@@ -90,6 +104,11 @@ public AppendSinkWriterBuilder(
         public AppendSinkWriter createWriter() {
             return new AppendSinkWriter(tablePath, flussConfig, tableRowType, ignoreDelete);
         }
+
+        @Override
+        public DataStream<RowData> addPreWriteTopology(DataStream<RowData> input) {
+            return input;
```

Comment:
should also add bucket shuffle if it has bucket key?

Reply:
Done!